src/PhpSpec/Console/IO.php

```diff
@@ -342,4 +342,25 @@ public function getBlockWidth()
         }
         return $width;
     }
+
+    /**
+     * @param string $message
+     * @param int|null $indent
+     */
+    public function writeError($message, $indent = null)
+    {
+        $message = wordwrap($message, $this->getBlockWidth() - 4, "\n", true);
```

Comment:
to be consistent with `askConfirmation()` https://github.com/phpspec/phpspec/blob/master/src/PhpSpec/Console/IO.php#L274